UITests/DomainAutocompleteTests.swift

```diff
@@ -72,6 +80,10 @@ class DomainAutocompleteTests: KIFTestCase {
     }
 
     private func ensureAutocompletionResult(textField: UITextField, prefix: String, completion: String) {
+        tester().runBlock({ (err) -> KIFTestStepResult in
```

Comment:
Maybe add a comment here that says we need to wait for the text entered callback due to debouncing?